src/Message/DocumentMessage.php

```diff
@@ -31,7 +31,7 @@ final class DocumentMessage extends Message
     /**
     * {@inheritdoc}
     */
-    public function __construct(string $to, MediaID $id, string $name, ?string $caption, ?string $reply_to)
+    public function __construct(string $to, MediaID $id, string $name, ?string $caption = '', ?string $reply_to)
```

Comment:
This line change does not make sense if "reply_to" does not have a default value as well.

I would leave this part of the code intact since the change made in the WhatsApp superclass is enough.

Reply:
Sorry for late replay, I have removed it. Thanks